contributions.md

```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
 4. Additional content describing common pitfalls, challenges, or limitations teams commonly encounter when applying certain practices or capabilities.
 5. Typos or grammatical fixes.
 
-If you chose to contribute a new practice try to follow our pre-established structure:
+If you chose to contribute a new practice, please try to follow our pre-established structure:
 
 ```markdown
-# Title Of Practice
+# Title of Practice
 
-A 2-3 sentence description of the practice followed by a sentence or two describing the main benefits of adopting this practice. Try to keep it brief.
+A 2-3 sentence description of the practice, followed by a sentence or two describing the main benefits of adopting this practice. Try to keep the text brief, relatable, and motivational.
 
 ## Nuance
 
-This section outlines 2-4 common pitfalls, challenges, or limitations teams commonly encounter when adopting this practice. Each nuance should have a subheader and description.
+This next section outlines 2-4 common pitfalls, challenges, or limitations that teams commonly encounter when adopting this practice. Each nuance should have a title (styled as a subhead) and a brief (3-5 sentences) description.
 
-## How To Improve
+*Begin this section with the following text:* "This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this practice. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the practice with your team(s)."
 
-List practices from [Learning Culture](/capabilities/learning-culture.md) and add the practice specific information (talking points, instructions, links to external resources, etc) in the description. See the How To Improve sections in [other practice pages](/practices/) for examples.
+## Putting the Practice to Use [or "Practice in Action"?]
+
+*Begin this section with the following text:* "The following actions will help your team implement this practice." 
```

Comment:
OK? Other ideas for the boilerplate text: 
    - “This section elaborates on general practices, giving you specific and actionable ways to put this practice to use.” 
    - “This section provides specific and actionable ideas for putting this practice to use.”